daras_ai_v2/asr.py

```diff
@@ -440,6 +429,30 @@ def asr_language_selector(
     )
 
 
+def get_language_match(lang: str | None, languages: list[str]) -> str | None:
```

Comment:
why not use `normalised_lang_in_collection` ??

Reply:
My bad. Didn’t realize we had a function for this. I looked at the asr code and only saw the place were we manually did it inline without calling a function so thought we didn’t have this refactored yet

Reply:
[01e3c81](https://github.com/GooeyAI/gooey-server/pull/414/commits/01e3c81a1c3afe488e24192ca928ddc5728a289b)